### DISCUSSION TITLE
Azure AD redirects to wrong login page

### DISCUSSION THREAD

**Discussion by jhk-85** [requarks/wiki#3565] [Error / Bug Report]
**Describe the bug**
Authorizing through Azure AD redirects to wrong login domain.

**To Reproduce**
1. Configure the Azure AD authentication strategy
2. Logout and navigate to the login view
3. Click on the Azure AD icon
4. The site redirects to `https://my.wiki.domain/[...]` instead of `https://login.microsoftonline.com/[...]`

**Expected behavior**
You should be redirected to the correct Azure AD sign-in site.

**Additional info***
If I replace `my.wiki.domain` name in the url to `login.microsoftonline.com`, I can succesfully login.

**Host Info:**
 - OS: Windows Server 2019
 - Wiki.js version: 2.0.0-beta.241
 - Database engine: Microsoft SQL Server 2008 R2

**Comment by keagan-ladds** [comment]
@jhk-85 Are you reverse proxying Wiki.js using IIS? I have just come across the same issue while using IIS.

**Comment by musashino205** [comment]
I faced the same problem with GitHub authentication in my environment,

```
Windows Server 2012 R2 (IIS 8.5) ---> Ubuntu Server 18.04.3 (Wiki.js)
```

 but solved by unchecking ```Reverse rewrite host in response headers``` in IIS Manager.
(server configuration (not virtual host) -> "Application Request Routing Cache" -> "Server Proxy Settings")
